pkg/mock/mockcluster/mockcluster.go

```diff
@@ -535,7 +540,7 @@ func (mc *Cluster) UpdateStoreStatus(id uint64) {
 }
 
 func (mc *Cluster) newMockRegionInfo(regionID uint64, leaderStoreID uint64, followerStoreIDs ...uint64) *core.RegionInfo {
-	return mc.MockRegionInfo(regionID, leaderStoreID, followerStoreIDs, []uint64{}, nil)
+	return mc.MockRegionInfo(regionID, &leaderStoreID, followerStoreIDs, []uint64{}, nil)
```

Comment:
When does `nil` need to be passed in here? In addition, `0` can also mean that store does not exist.

Reply:
I think check 0 is better.